pkg/service/roommanager.go

```diff
@@ -556,7 +556,7 @@ func (r *RoomManager) rtcSessionWorker(room *rtc.Room, participant types.LocalPa
 	_ = r.refreshToken(participant)
 	tokenTicker := time.NewTicker(tokenRefreshInterval)
 	defer tokenTicker.Stop()
-	stateCheckTicker := time.NewTicker(time.Millisecond * 50)
+	stateCheckTicker := time.NewTicker(time.Millisecond * 250)
```

Comment:
In a test case that has thousands of participants, the timer costs nearly 5% total CPU time, so increasing the interval to check the status

Reply:
Wow, good catch. Guessing this can be reduced even more, like once in 1/2 second. But, we can leave it 250ms for now.